draft-ietf-taps-interface.md

```diff
@@ -1049,26 +1048,47 @@ Type:
 : Enumeration
 
 Default:
-: Disabled
+: Disabled for connections created through initiate and rendezvous, Passive for listeners
 
-This property specifies whether an application wants to take advantage of
-transferring data across multiple paths between the same end hosts. Using
-multiple paths allows connections to migrate between interfaces as
-availability and performance properties change. Possible values are:
+This property specifies whether and how applications want to take advantage of
+transferring data across multiple paths between the same end hosts.
+Using multiple paths allows connections to migrate between interfaces or aggregate bandwidth
+as availability and performance properties change.
+Possible values are:
 
 Disabled:
-: The connection will not attempt using multiple paths once established
+: The connection will not allow using multiple paths once established and will disable respective functionality in case the chosen transport supports it.
```

Comment:
```suggestion
: The connection will not use multiple paths once established, even if the chosen transport supports using multiple paths.
```

Reply:
Okay, sorry, but I must ask: how do you people do this? I can't find how to make such direct text change proposals